### PR TITLE
chore(Dockerfile): update kubectl version to 1.19.3 and add .dockerignore

### PR DESCRIPTION
**Pull request by muzi502**
…nore file

**What type of PR is this?**

**What this PR does / why we need it**:

- update kubectl version to 1.19.3 for cluster-deploy-job image
- add `.dockerignore` file for don't copy useless or sensitive file to imag , such as `.github` dir and `OWNERS` file

**Which issue(s) this PR is related to** *(optional, link to 3rd issue(s))*:

Fixes #https://github.com/caicloud/kubespray/pull/34

Reference to #

**Special notes for your reviewer**:

/cc @bbbmj @WangMinjie114 

**Does this PR introduce a user-facing change?**:

```release-note

```

### CODE REVIEW

**Review by bbbmj — COMMENTED** [review]
/approve 
cc @WangMinjie114

File: .dockerignore [hunk 1]
```diff
@@ -0,0 +1,23 @@
+.ansible-lint
+.editorconfig
+.git
+.github
+.gitignore
+.gitlab-ci
```
Comment (WangMinjie114) [inline .dockerignore:6]: 这个 ci 的配置在镜像里去掉了，会不会影响到后面 kubespray 的 ci 的使用

Comment (muzi502) [reply .dockerignore:6]: gitlab-ci 的代码类似于 Jenkinsfile ，容器里不会用到这部分代码

**Pull request merged**